pages/exercises/[lessonSlug].tsx

```diff
@@ -170,19 +172,20 @@ const ExerciseList = ({
         </NewButton>
       </div>
       <div className="container">
-        <div className="row">
-          {mockExercisePreviews.map((exercisePreview, i) => (
-            <ExercisePreviewCard
-              key={i}
-              moduleName={exercisePreview.moduleName}
-              state={exercisePreview.state}
-              problem={exercisePreview.problem}
-              className={`col ${
-                i < mockExercisePreviews.length - 1 ? 'me-4' : ''
-              }`}
-            />
-          ))}
-        </div>
+        {chunk(mockExercisePreviews, 3).map((exercisePreviewChunk, i) => (
```

Comment:
This works well for 1000px width screens and above. We'll need to handle smaller screens in the future.